docs/mcd.md

```diff
@@ -1,14 +1,14 @@
-# Modular Connection Dialog (MCD)
+# Tableau Connector
```

Comment:
This is too generic. Maybe "Connection Dialogs v2"?

Reply:
Will make the change.